configs/mot/ocsort/ocsort_yolox_x_crowdhuman_mot17-private-half.py

```diff
@@ -0,0 +1,168 @@
+_base_ = [
+    '../../_base_/models/yolox_x_8x8.py',
+    '../../_base_/datasets/mot_challenge.py', '../../_base_/default_runtime.py'
+]
+
+img_scale = (800, 1440)
+samples_per_gpu = 1
```

Comment:
What's the default training setting here? 

8x1 seems not make sense to me. By default we support distributed training with 8 GPUs. If not, better clarify. 

The default config of ByteTrack uses `samples_per_gpu = 4`. Better to be consistent.

Reply:
Fixed.